genai_cookbook/nbs/4-evaluation-metrics.md

```diff
@@ -1,17 +1,17 @@
 ## Assessing performance: Metrics that Matter
 
-With an evaluation set, you are able to measure the performance of your RAG application across a number of different dimensions, including:
+With an evaluation set, you are able to measure the performance of your Agent application across a number of different dimensions, including:
```

Comment:
Lowercase "agent" here too - also maybe we can look for instances of "A agent" and replace with "An agent"